contrib/package/nix/copyparty/default.nix

```diff
@@ -101,8 +97,7 @@ stdenv.mkDerivation {
     homepage = "https://github.com/9001/copyparty";
     changelog = "https://github.com/9001/copyparty/releases/tag/v${pinData.version}";
     license = lib.licenses.mit;
-    inherit (python.meta) platforms;
     mainProgram = "copyparty";
-    sourceProvenance = [ lib.sourceTypes.binaryBytecode ];
+    sourceProvenance = [ lib.sourceTypes.fromSource ];
```

Comment:
By the way this is implicitly the default when no `sourceProvenance` was specified.